docs/elements.md

```diff
@@ -1723,6 +1723,7 @@ def generate(data):
 | `negative-weights`          | boolean | false              | Whether to recognize negative weights in an adjacency matrix. If set to false, then all weights at most 0 are ignored (not counted as an edge). If set to true, then all weights that are not `None` are recognized.                                                    |
 | `directed`                  | boolean | true               | Whether to treat edges in an adjacency matrix as directed or undirected. If set to false, then edges will be rendered as undirected. _The input adjacency matrix must be symmetric if this is set to false._                                                            |
 | `weights-presentation-type` | string  | `'f'`              | Number display format for the weights when using an adjacency matrix. If presentation-type is 'sigfig', each number is formatted using the to_precision module to digits significant figures. Otherwise, each number is formatted as `{:.{digits}{presentation-type}}`. |
+| `show-warnings`             | boolean | true               | If set to true, creates issues from warnings raised by Graphviz rendering. Otherwise, hides these warnings.                                                                                                                                                             |
```

Comment:
Maybe call this `log-warnings` instead? Let's also not mention issue creation here; the fact that logs create issues is not guaranteed to be true forever.

Reply:
That makes sense, and I think the name `log-warnings` is clearer.

Reply:
Done here: https://github.com/PrairieLearn/PrairieLearn/pull/7367/commits/eff1854cbbb900f734e6e7ef9c6d353c03734327